tasks/update-infra-deployments/README.md

```diff
@@ -18,6 +18,8 @@
 | sharedSecret            | Secret in the namespace which contains private key for the GitHub App                        | true     | infra-deployments-pr-creator                                                                                                                     |
 | githubAppID             | ID of Github app used for updating PR                                                        | true     | 305606                                                                                                                                           |
 | githubAppInstallationID | Installation ID of Github app in the organization                                            | true     | 35269675                                                                                                                                         |
+## Changes since 0.4.1
```

Comment:
Can you add a new line before this line?

Reply:
Changes should be visible now